.github/workflows/release.yml

```diff
@@ -0,0 +1,103 @@
+name: Release
+on:
+  workflow_dispatch:
+    inputs:
+      increment:
+        description: 'SemVer Increment'
+        required: true
+        default: 'prerelease'
+        type: choice
+        options:
+        - prerelease
+        - prepatch
+        - preminor
+        - premajor
+        - patch
+        - minor
+        - major
+      identifier:
+        description: 'npm Tag'
+        required: true
+        default: 'canary'
+        type: choice
+        options:
+        - canary
+        - latest
+jobs:
+  build:
+    runs-on: macos-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+          token: ${{ secrets.TURBOBOT }}
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.18.0
+          cache: true
+          cache-dependency-path: cli/go.sum
+
+      - name: Set Up Protoc
+        uses: arduino/setup-protoc@v1
+        with:
+          version: "3.x"
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set Up Go and GRPC protobuf
+        run: |
+          go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.28.0
+          go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.2.0
+
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v3
+        with:
+          # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
+          version: latest
+
+          # Optional: working directory, useful for monorepos
+          working-directory: cli
+
+          # Optional: golangci-lint command line arguments.
+          args: --issues-exit-code=0
```

Comment:
Why are we running this if we're continuing on errors?

Reply:
I'm using this to make sure that the installation of `golangci-lint` matches our normal CI setup. `golangci-lint`'s action does not have an install-only option like GoReleaser, so I'm turning the auto-triggered run into a no-op. During `make publish` that _also_ triggers linting and we do not ignore the outcome (lint failure prevents release).